rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -2691,6 +2750,8 @@ void when_registerBtcTransaction_usesLegacyType_beforeFork_lock_and_no_refund()
         // Assert
         co.rsk.core.Coin totalAmountExpectedToHaveBeenLocked = co.rsk.core.Coin.fromBitcoin(amountToLock);
 
+        whitelist.consume(btcAddress);
```

Comment:
Why manually consume it?

Reply:
Done! Removed manually whitelist address consumption since it's not needed anymore.